recipes/ninja-hep-ph/meta.yaml

```
@@ -0,0 +1,138 @@
+{% set name = "ninja" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name }}-hep-ph
+  version: {{ version }}
+
+source:
+  url: https://github.com/peraro/ninja/archive/v{{ version }}.tar.gz
+  sha256: b9709aada408286b6533ac7427bab9d1364d449a9c11098c324ea0875fe9a9e1
+  patches:
+    - posix-compliant-shell.patch
+
+build:
+  # libquadmath is required, so macOS is not supported
+  skip: true  # [not linux]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('ninja-hep-ph', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - automake  # includes autoconf
+    - libtool
+    - make
+    - file
+    - pkg-config
+    - gnuconfig
+    - oneloop  # [build_platform != target_platform]
+    - looptools-static  # [build_platform != target_platform]
+  host:
+    - oneloop
+    - looptools-static
+  run:
+    - oneloop
+    - looptools-static
+
+test:
+  # As need to regenerate Makefiles, need all of the source files that
+  # configure uses to do so
+  source_files:
+    - examples/
+    - m4/
+    - src/
+    - Makefile.am
+    - configure.ac
+  requires:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - automake
+    - libtool
+    - make
+  commands:
+    - test -f $PREFIX/lib/libninja${SHLIB_EXT}
+    # c.f. https://github.com/conda/conda-build/issues/3075
+    - test ! -f $PREFIX/lib/libninja.la
+    - test -f $PREFIX/lib/libninja.a
```

```suggestion
    - test ! -f $PREFIX/lib/libninja.a
```

I guess this should get removed given

> Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).